doc/releases/changelog-dev.md

```diff
@@ -76,12 +76,17 @@
 * `StatePrep` operations expanded onto more wires are now compatible with backprop.
   [(#5028)](https://github.com/PennyLaneAI/pennylane/pull/5028)
 
+* `import_state` now applies the chemist-to-physicist sign convention switch when
```

Comment:
might want to add a link for `chemist-to-physicist` sign convention.

Reply:
Couldn't find a simple link to this convention, but expanded the explanation and also comments within the functions themselves to make it clear (also the docstring of _sign_chem_to_phys` has a more in-depth explanation).